test/extractor/CaretValueRuleExtractor.test.ts

```diff
@@ -59,6 +65,16 @@ describe('CaretValueRuleExtractor', () => {
       expect(caretRules).toEqual<ExportableCaretValueRule[]>([expectedRule]);
     });
 
+    it('should always extract a caret value rule for status when it is not "active"', () => {
+      const sd = cloneDeep(looseBSSD);
+      sd.status = 'draft';
```

Comment:
Do you need to clone `looseBSSD` and set the status to `'draft'`?  Isn't it already draft?

Reply:
Not really, I did purposefully set it as 'draft' just so that someone who was looking at the test could see what was going on without having to look at the definition.

Reply:
But it is redundant, and I don't mind removing it if you don't like it.

Reply:
Eh.  It's fine.  Let's just get it merged!